### PR TITLE
Added providers for locale ka_GE

### PR DESCRIPTION
**Pull request by akalongman**
Added Address, Color, DateTime, Internet and PhoneNumber providers for locale ka_GE

### CODE REVIEW

File: src/Faker/Provider/ka_GE/Address.php [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    }
+
+    public function address()
+    {
+        $format = static::randomElement(static::$addressFormats);
+
+        return $this->generator->parse($format);
+    }
+
+    public static function country()
```
Comment (fzaninotto) [inline src/Faker/Provider/ka_GE/Address.php]: No need to override methods of the parent class when it's the same code

**Comment by fzaninotto** [comment]
Build fails, see the CI report.

**Comment by akalongman** [comment]
@fzaninotto build fails, buts its not script bug, look at Travis report https://travis-ci.org/fzaninotto/Faker/jobs/91271242

**Comment by fzaninotto** [comment]
True, I've restarted the build.

**Comment by akalongman** [comment]
@fzaninotto what about this PR?

**Comment by akalongman** [comment]
@fzaninotto PR updated

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**